docs/examples/pvfleets-qa-pipeline/pvfleets-irradiance-qa.py

```diff
@@ -0,0 +1,375 @@
+"""
+PV Fleets QA Process: Irradiance
+================================
+
+PV Fleets Irradiance QA Pipeline
+"""
+
+# %%
+# The NREL PV Fleets Data Initiative uses PVAnalytics routines to assess the
+# quality of systems' PV data. In this example, the PV Fleets process for
+# assessing the data quality of an irradiance data stream is shown. This
+# example pipeline illustrates how several PVAnalytics functions can be used
+# in sequence to assess the quality of an irradiance data stream.
+
+import pandas as pd
+import pathlib
+from matplotlib import pyplot as plt
+import pvanalytics
+import pvlib
+from pvanalytics.quality import data_shifts as ds
+from pvanalytics.quality import gaps
+from pvanalytics.quality.outliers import zscore
+from pvanalytics.features.daytime import power_or_irradiance
+from pvanalytics.quality.time import shifts_ruptures
+from pvanalytics.features import daytime
+
+# %%
+# First, we import a POA irradiance data stream from a PV installation
+# at NREL. This data set is publicly available via the PVDAQ database in the
+# DOE Open Energy Data Initiative (OEDI)
+# (https://data.openei.org/submissions/4568), under system ID 15.
+# This data is timezone-localized.
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+file = pvanalytics_dir / 'data' / 'system_15_poa_irradiance.csv'
+time_series = pd.read_csv(file, index_col=0, parse_dates=True).squeeze()
+latitude = 39.7406
+longitude = -105.1775
+data_freq = '15T'
+time_series = time_series.asfreq(data_freq)
+
+# %%
+# First, let's visualize the original time series as reference.
+
+time_series.plot(title="Original Time Series")
+plt.show()
+
+# %%
+# Now, let's run basic data checks to identify stale and abnormal/outlier
+# data in the time series. Basic data checks include the following steps:
+#
+# 1) Flatlined/stale data periods (:py:func:`pvanalytics.quality.gaps.stale_values_round`)
+# 2) Negative irradiance data
+# 3) "Abnormal" data periods, which are defined as less than 10% of the
+#    daily time series mean OR greater than 1300
+# 4) Outliers, which are defined as more than one 4 standard deviations
+#    away from the mean (:py:func:`pvanalytics.quality.outliers.zscore`)
+
+# 1) REMOVE STALE DATA (that isn't during nighttime periods)
+# Day/night mask
+daytime_mask = power_or_irradiance(time_series)
+# Stale data mask
+stale_data_mask = gaps.stale_values_round(time_series,
+                                          window=3,
+                                          decimals=2)
+stale_data_mask.loc[(stale_data_mask) &
+                    (~daytime_mask)] = False
+
+# 2) REMOVE NEGATIVE DATA
+negative_mask = (time_series < 0)
+
+# FIND ABNORMAL PERIODS
+daily_min = time_series.resample('D').min()
+erroneous_mask = (daily_min < 50)
```

Comment:
This is unfortunate, would be more consistent if `True` meant good data. Maybe `daily_min > 50`

Reply:
updated in https://github.com/pvlib/pvanalytics/pull/202/commits/d3c2d56f7b7ced1c06cc9da4af494cb38fddc84c